python/katana/local/_graph.pyx

```diff
@@ -301,6 +303,14 @@ cdef class GraphBase:
         types = manager.GetAtomicEntityTypeIDs()
         return [EntityType.make(manager, typeid) for typeid in types]
 
+    @abstractmethod
+    def global_degree(self, uint64_t node):
```

Comment:
Shouldn't this be `out_degree`? It's not the total degree.

Reply:
Works for me.